src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -317,7 +317,8 @@ internal sealed class RSession : IRSession, IRCallbacks {
             // Try graceful shutdown with q() first.
             if (host != null) {
                 try {
-                    await Task.WhenAny(hostRunTask, host.QuitAsync(), Task.Delay(500)).Unwrap();
+                    host.QuitAsync().DoNotWait();
```

Comment:
Need to silence exceptions on this. I think it's safe to silence everything here.